lxd/instance/drivers/driver_qemu.go

```diff
@@ -2813,6 +2816,11 @@ cp systemd/lxd-agent.service /lib/systemd/system/
 cp systemd/lxd-agent-setup /lib/systemd/
 systemctl daemon-reload
 
+# SELinux handling.
+if getenforce >/dev/null 2>&1; then
+    semanage fcontext -a -t bin_t /run/lxd_agent/lxd-agent
```

Comment:
And is `/run/lxd_agent/lxd-agent` where the lxd-agent is actually placed?

Reply:
Yes, it is placed in `/run/lxd_agent.lxd-agent`
```
$ ls -lah /run/lxd_agent/lxd-agent 
-r-x------ 1 root root 17M Feb 13 14:11 /run/lxd_agent/lxd-agent
```

As @simondeziel pointed out, on this path we mount the agent in the [setup script](https://github.com/canonical/lxd/blob/main/lxd/instance/drivers/driver_qemu.go#L2731-L2733), and [systemd unit](https://github.com/canonical/lxd/blob/main/lxd/instance/drivers/driver_qemu.go#L2706-L2721) also uses that path.